### PR TITLE
Feat: add support for search in docs

### PR DESCRIPTION
**Pull request by ovaistariq**

### CODE REVIEW

**Comment by ovaistariq** [comment]
Appears like this in the UI 
image

File: docusaurus.config.js [hunk 1]
```diff
@@ -156,6 +156,12 @@ const config = {
         darkTheme: darkCodeTheme,
         additionalLanguages: ["java", "scala"],
       },
+      algolia: {
```
Comment (adilansari) [inline docusaurus.config.js:159]: :(

**Review by adilansari — APPROVED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (adilansari) [reply docusaurus.config.js:159]: Fyi - https://github.com/typesense/docusaurus-theme-search-typesense

**Review by rbarabas — APPROVED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (ovaistariq) [reply docusaurus.config.js:159]: Yeah I looked at it but it requires setting up some infra. DocSearch for docusaurus is very simple as you can see in this diff.

**Pull request merged**